config/routes.rb

```diff
@@ -114,6 +114,7 @@
   scope via: :all do
     match "/400" => "errors#bad_request"
     match "/404" => "errors#not_found"
+    match "/403" => "errors#forbidden"
```

Comment:
Nitpick: This should probably be in numerical order.

Reply:
D'oh! the shame